lms/djangoapps/certificates/api.py

```diff
@@ -155,6 +146,19 @@ def get_certificate_for_user(username, course_key, format_results=True):
         return cert
 
 
+def get_certificate_for_user_id(user, course_id):
```

Comment:
We already have an API function to retrieve a certificate object with the `get_certificate_for_user` function. To be honest, I kind of hate that function now (`get_certificate_for_user`) and it is my fault why I hate it (forgive me), but I wanted to point out that it exists.

Might be worth keeping your function and migrating some of our code to using it in the future. At least that way `get_certificate_for_user` can go back to having a singular purpose.

Reply:
I had to write this newer version because I needed to get the data based on the user object and not the username. It also seemed like 1 less query. Are you asking me to make a change or just highlighting the change?

Reply:
Ah, I am following now, I didn't pick up on that difference. Thanks for pointing that out. I'm not asking for a change at this time.